### ISSUE TITLE
Travis CI Post mortems

### ISSUE THREAD

**Issue by andygrunwald** [danluu/post-mortems#24]
[TravisCI](https://travis-ci.org) has several post mortems as well:

* [Post-Mortem: Build Outage and Security Advisory](http://blog.travis-ci.com/2013-01-31-post-mortem-build-outage/)
* [Post-Mortem: Pull Request Unavailability](http://blog.travis-ci.com/2012-09-24-post-mortem-pull-request-unavailability/)
* [Post Mortem: Recent Build Infrastructure Issues](http://blog.travis-ci.com/2013-04-26-post-mortem-build-infrastructure-issues/)

Maybe they are worth to add? :)

**Comment by andygrunwald** [comment]
@danluu any update here? Anything i can do? Adding them as a PR? Or won`t them accepted? Then i will close this issue here.

**Comment by danluu** [comment]
Oh, sorry, I've been pretty slow about these. If you want to add a PR, I'd be happy to merge it. Otherwise, I'll write these up whenever I get some spare time :-).

**Comment by DavidTPate** [comment]
Yeah, throw up a PR @andygrunwald That's how most people contribute that I've seen.
